### ISSUE TITLE
When opening a onenote book, it redirects to the browser

### ISSUE THREAD

**Issue by GM89** [anujdatar/onenote-desktop#31]
When I try to open one of my other OneNote Books, it redirects me to the browser. 
Therefore I can only work on my notebooks through the browser, not your app. Therefore I can't use this app. 
I have Ubuntu 20.04.3 LTS, and gnome 3.36.
Could I ask if there are plans to solve this bug?
Thanks

**Comment by anujdatar** [comment]
Hi, do you see this in the latest build of the? `v2.0.0`? I've had a fix for this behavior since v1.3, and did not see this behavior during my limited testing.

I had to change the implementaion in the latest version because of changes with electron and typescript. Let me see what I can do.

If I recall correctly, a new window only pops-up when you click on the small dropdown menu while you are in another notebook. 

While inconvenient, for now, you could navigate to `OneNote Home` using the app menubar at the top. And then clicking on the notebook you want should open it inside the app. Temporary work around, but I'll work on a fix over then next day or two.

Thanks for letting me know.

**Comment by GM89** [comment]
I use v 2.0.0.  
Alright, I'll use OneNote Home in the navbar to avoid this error.  
Thanks for your time

**Comment by anujdatar** [comment]
Jjust released a fix for the issue, v2.0.1 should fix this issue. If you are using the snap package, it should automatically update itself in a soon enough.

Do let me know if it does not work for you. OneNote does this weird thing where it tries to access some newly created notebooks through OneDrive. That seemed to be the cause of the issue. I added one new route that I saw to the list of exceptions. Hopefully they aren't using too many of these domains.

If it does not fix the issue for you, I might have to implement a dialog/popup tbat asks you where to open these links.

Thanks
